src/__test__/components/data-management/ProjectDetails.test.jsx

```diff
@@ -234,26 +234,6 @@ describe('ProjectDetails', () => {
     expect(metadataButton).not.toBeDisabled();
   });
 
-  it('Shows empty data if there is no data', () => {
```

Comment:
This test is no longer relevant -- if there is no data, ProjectDetails component will never render (see changes to index.jsx file)